constants.py

```diff
@@ -0,0 +1 @@
+PROCESSOR_VERSION = "0.2.0"
```

Comment:
I think this should be set when a release is made, but let's suppose we will create it soon

Reply:
I mean that technically we are still 0.1.0

Reply:
That's true, we could include the changelog in this pull request